packages/rancher-istio/1.15/rancher-kiali-server/generated-changes/patch/values.yaml.patch

```diff
@@ -18,12 +18,12 @@
 +  repository: rancher/mirrored-kiali-kiali
    image_pull_policy: "Always"
    image_pull_secrets: []
--  image_version: v1.52.0 # version like "v1.39" (see: https://quay.io/repository/kiali/kiali?tab=tags) or a digest hash
+-  image_version: v1.57.3 # version like "v1.39" (see: https://quay.io/repository/kiali/kiali?tab=tags) or a digest hash
 +  tag: v1.52.0 # version like "v1.39" (see: https://quay.io/repository/kiali/kiali?tab=tags) or a digest hash
```

Comment:
Should this version be v1.57.3? (I'm not familiar with istio, it just seems odd)

Reply:
Ah! my bad. That should be 1.57.3. Updating the PR.